example/02-kademlia/rendezvous_chat.cpp

```diff
@@ -175,7 +179,11 @@ namespace {
 int main(int argc, char *argv[]) {
   // prepare log system
   auto logging_system = std::make_shared<soralog::LoggingSystem>(
-      std::make_shared<libp2p::log::Configurator>(logger_config));
+      std::make_shared<soralog::ConfiguratorFromYAML>(
```

Comment:
Logger initialization looks a bit wordy now.

Reply:
No, it's correct. It is needed to apply libp2p's part of config before app's part